src/ampycloud/data.py

```diff
@@ -93,16 +94,31 @@ def _cleanup_pdf(self, data: pd.DataFrame) -> pd.DataFrame:
         # Begin with a thorough inspection of the dataset
         data = utils.check_data_consistency(data, req_cols=self.DATA_COLS)
 
+        # By default we set NSC to false, NOTE: attrs is still experimental in pandas!
+        try:
+            data.attrs['high_clouds_detected'] = False
+        except AttributeError:
+            # NOTE: Untestable as we cannot monkeypatch attrs as it would result in side-effects for __repr__
+            # which break other tests
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore") # pandas assumes we are trying to assign a column, which is not the case
+                data.attrs = {'high_clouds_detected': False}
+
```

Comment:
Interesting approach ! I like it, in that it keeps this very close to the data. Without thinking too much about it, I guess I would have implemented it as a class property (e.g. like self.msa). But if you're ok with this approach, then it is certainly ok with me.

Reply:
I'm okay with it and prefer to have the flag close to the data (reason: I think the CeiloChunk class is quite big and versatile already, we might want to split it up at some point and it will come in handy if there are not more attributes).